README.md

```diff
@@ -886,35 +886,59 @@ service. However, support is available to manually enable those services. To do
+balancer-terminated SSL connection. 
+
+Provide a 
+[`ECOMMERCE_PAYMENT_PROCESSOR_CONFIG`](https://github.com/edx/configuration/blob/d68bf51d7b8403bdad09dc764af5ebafe16d7309/playbooks/roles/ecommerce/defaults/main.yml#L103)
+which contains the payment processors and their keys.  Note that if you want to
+run payment processors other than the defaults (check your version of
+`ecommerce.settings._oscar`) you may need to override more variables.
 
 Once the spawn is complete, you'll need to take the following steps to finish setup
 (these are one-time actions that are stored in the database):
 
-1. Update the staff user so that a first name is associated with it
-   (some routines look for a user profile).
-2. Update the database-backed OAuth2 clients for course discovery and
-   ecommerce so that they're associated with the staff user.
-3. In the sandbox ecommerce env, configure the Enterprise Partner, Site,
-   and SiteConfiguration [per the instructions on this page](http://edx.
-   readthedocs.io/projects/edx-installing-configuring-and-running/en/lat
-   est/ecommerce/install_ecommerce.html#configure-a-site-partner-and-sit
-   e-configuration).
-4. In the sandbox discovery env, configure a partner [as described here]
-   (https://open-edx-course-catalog.readthedocs.io/en/latest/getting_sta
-   rted.html#configure-partners).
+1. Create/choose a staff user to use for the OAuth2 clients.
+   Ensure the staff user has a user profile associated (i.e. set a Full Name).
+1. In Django Admin > OAuth2 > Clients, there should already be clients created
+   for ecommerce and discovery.  
+   If not, [create and register new clients](http://edx.readthedocs.io/projects/edx-installing-configuring-and-running/en/latest/ecommerce/install_ecommerce.html#configure-edx-openid-connect-oidc)
+   for each service.  You'll need the client IDs and client secrets for the next
+   steps.
+   Ensure that both clients are attached to the staff user updated above.
+1. In the ecommerce env, add a Site, Partner, and Site Configuration as per the
+   instructions in the [edX ecommerce docs](http://edx.readthedocs.io/projects/edx-installing-configuring-and-running/en/latest/ecommerce/install_ecommerce.html#add-another-site-partner-and-site-configuration).
+1. In the discovery env, configure a partner using
+   [`create_or_update_partner`](https://github.com/edx/course-discovery/blob/master/course_discovery/apps/core/management/commands/create_or_update_partner.py).
+   Use the same partner code as what you used for ecommerce.
+1. [Configure LMS to use ecommerce](http://edx.readthedocs.io/projects/edx-installing-configuring-and-running/en/latest/ecommerce/install_ecommerce.html#switch-from-shoppingcart-to-e-commerce)
+
+Test your configuration:
+
+1. Verify the OIDC login for both services:
+    https://ecommerce-<your-instance>.opencraft.hosting/login
+    https://discovery-<your-instance>.opencraft.hosting/login
+1. Verify that the discovery cronjob runs without errors:
```

Comment:
Does this need to be manually set up as a cron job too? I didn't find where in configuration the cron job is added

Reply:
True..  It's in our `opencraft-release/ginkgo.1` playbook, but unfortunately hasn't been contributed (or wasn't accepted?) upstream.  I've added a note about it https://github.com/open-craft/opencraft/pull/294/commits/efab883dbcd784a4950ee15737be7a8b9e79e363#diff-04c6e90faac2675aa89e2176d2eec7d8R887

Reply:
It's also not in the [opencraft-release/ginkgo.1-cloudera branch](https://github.com/open-craft/configuration/blob/opencraft-release/ginkgo.1-cloudera/playbooks/roles/discovery/tasks/main.yml).